pkg/kritis/cryptolib/verifier.go

```diff
@@ -67,11 +68,16 @@ type jwtVerifier interface {
 	verifyJwt(signature []byte, publicKey []byte) ([]byte, error)
 }
 
+type authenticatedAttFormer interface {
```

Comment:
`authenticatedAttConverter`?

Reply:
Done. I removed this interface and replaced with a function that will be supplied to `checkAuthenticatedAttestation`. That way we can test the behavior of `checkAuthenticatedAttestation` separately from the conversion functionality